### PR TITLE
Add Samsung Knox Asset Intelligence Sentinel Solution

### PR DESCRIPTION
**Pull request by sean-mcclelland**
Required items, please complete
   
   Change(s):
   - Add Samsung Knox Asset Intelligence Sentinel Solution

   Reason for Change(s):
   - PR for adding the new Samsung Knox Asset Intelligence Sentinel Solution

   Version Updated:
   -  N/A -- First version

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by sean-mcclelland** [comment]
@microsoft-github-policy-service agree company="Samsung"

**Comment by v-prasadboke** [comment]
Hello @sean-mcclelland, Thanks for raising this PR. This PR will be investigated and we will provide you an update about the same before 24 December, 2024

**Comment by v-prasadboke** [comment]
I see this PR has CCP connector. Please refer this connector and readme for CCP files arrangement and mapping 
[Connector](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Ermes%20Browser%20Security/Data%20Connectors/ErmesBrowserSecurityEvents_ccp)

[readme](https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/V3/CCP_README.md)

File: Solutions/Samsung Knox Asset Intelligence/SolutionMetadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+	"publisherId": "samsungelectronics1734042706970",
+	"offerId": "samsung-knox-asset-intelligence-sentinel",
```
Comment (v-prasadboke) [inline Solutions/Samsung Knox Asset Intelligence/SolutionMetadata.json:3]: please include azure-sentinel in offerid

File: Solutions/Samsung Knox Asset Intelligence/Analytic Rules/SamsungKnoxMobileDeviceBootCompromise.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  | where Name == "BOOT_COMPROMISED_SOFTWARE_BINARY"
+  and MitreTtp has "T1645"
+suppressionEnabled: false
+suppressionDuration: 5h
+incidentConfiguration:
+  createIncident: true
+  groupingConfiguration:
+    enabled: false
+    reopenClosedIncident: false
+    lookbackDuration: 5h
```
Comment (v-prasadboke) [inline Solutions/Samsung Knox Asset Intelligence/Analytic Rules/SamsungKnoxMobileDeviceBootCompromise.yaml:28]: Please replace 5h with 5H in all the analytic rules

**Comment by v-prasadboke** [comment]
Hello @sean-mcclelland, Can you please provide me write access to your branch

**Comment by v-prasadboke** [comment]
Also can you please tell me what kind of this Data connector is ?

I'm confused about it. 
When I deployed the data connector to portal, It was not visible

**Comment by sean-mcclelland** [comment]
> Also can you please tell me what kind of this Data connector is ?
> 
> I'm confused about it. When I deployed the data connector to portal, It was not visible

@v-prasadboke Thank you for your help reviewing this PR. This PR contains a REST API (Push to Microsoft Sentinel) based Data Connector so it is not exactly a CCP based approach. This product is for Samsung Knox Asset Intelligence customers and allows them to push security logs from their devices to their Azure Sentinel setups via the Samsung KAI Cloud Services. Customers set up their configuration directly in the KAI Service Portal after which logs are pushed via their provided DCE/Generated url. 

When it deploys you should see something similar to:
Screenshot 2024-12-23 at 2 56 34 PM

@jaspreet-saini will update this PR shortly with your two requested fixes.

File: Solutions/Samsung Knox Asset Intelligence/Analytic Rules/SamsungKnoxMobileDeviceBootCompromise.yaml (hunk 2 above)
Comment (sean-mcclelland) [reply Solutions/Samsung Knox Asset Intelligence/Analytic Rules/SamsungKnoxMobileDeviceBootCompromise.yaml]: This has been corrected now. Thanks!

File: Solutions/Samsung Knox Asset Intelligence/SolutionMetadata.json (hunk 1 above)
Comment (sean-mcclelland) [reply Solutions/Samsung Knox Asset Intelligence/SolutionMetadata.json]: The offerId field has been updated. Thanks!

**Comment by v-prasadboke** [comment]
Hello @sean-mcclelland, Can we get on a call. I have some doubts about the parameters declared in Data connectors. 
Please share your time zone and availability so that we can plan according the same. 

You can ping me directly teams as well at v-prasadboke@microsoft.com

**Review by v-prasadboke — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**